### ISSUE TITLE
Question for Loss Weight

### ISSUE THREAD

**Issue by LeslieKid** [ThinkMorph/ThinkMorph#4]
Great work!

The design for MSE and CE loss is great and I think it's a good idea to introduce loss weight to balance the potential numeric gap between MSE and CE loss. May I ask for why do you set both loss weight equal 1.0 ?

**Comment by hychaochao** [comment]
Good question! In fact, we initially experimented with different weights for the MSE loss and CE loss on visual search task (e.g., MSE weight = 1.7, CE loss = 0.3), but we found that the model trained this way performed similarly on the benchmark as if both were set to 1. Therefore, we set both to 1 in subsequent training. However, this is a hyperparameter worth exploring, and we look forward to further investigation!
